manifests/pg_repack.pp

```diff
@@ -15,13 +15,17 @@
 # @param activity_tables_repack_timer [String] The Systemd timer for the pg_repack job affecting the 'activity' tables
 # @param reports_tables_repack_timer [String] Deprecated Parameter will be removed in future releases
 # @param resource_events_tables_repack_timer [String] Deprecated Parameter will be removed in future releases
+# @param repack_run_level 
```

Comment:
These two parameters need a brief description of what they do in their `@param` strings.  I think `repack_log_level` would also be a better name for this one.

Reply:
Agree with `repack_log_level` being a better name. "log level" is a common term that sysadmins will recognize and understand.

Reply:
@Sharpie @m0dular No problem, I'll refactor these at once and push up.